### ISSUE TITLE
Improve nuget.org login flow

### ISSUE THREAD

**Issue by skofman1** [NuGet/NuGetGallery#7086]
This task is about analyzing the differences between our login flow to MSA/AAD vs. other websites like outlook.com and see what we can improve. For example: should nuget.org automatically recognize you are logged in if you are logged-in to your MSA/AAD account?

**Comment by weitzhandler** [comment]
> hould nuget.org automatically recognize you are logged in if you are logged-in to your MSA/AAD account?

Yes please. I always find myself logging in over and over whenever I visit NuGet. Please fix this. Once I log in, I should stay logged in for at least 3 weeks of inactivity, or forever. Or alternatively, depending on my MSA/AAD login.
